client/src/components/dashboard/Preferences.js

```diff
@@ -75,13 +75,14 @@ class Preferences extends React.Component {
       interestsOptions: [],
       skillsOptions: [],
     }
-    // SHARED ERRORS:
-    this.EMAIL_ERROR = "Please enter a valid email address."
-    this.LOCATION_ERROR = "Location must be 25 characters or less.";
-    this.DISPLAY_NAME_ERROR = "Display name must be 40 characters or less.";
-    this.CAREER_ERROR = "Please complete the entire section or clear the form."
-    this.CODEPEN_ERROR = "Please enter your username only, not your profile url.";
-    this.MENTORSHIP_ERROR = "To complete your mentorship prorgram enrollment, please fill out the section above.";
+    // SHARED VALIDATION ERRORS:
+    this.EMAIL_ERROR = 'Please enter a valid email address.';
+    this.LOCATION_ERROR = 'Location must be 25 characters or less.';
+    this.DISPLAY_NAME_ERROR = 'Display name must be 40 characters or less.';
+    this.CAREER_ERROR = 'Please complete the entire section or clear the form.';
+    this.CODEPEN_ERROR = 'Please enter your username only, not your profile url.';
+    this.COUNTRY_ERROR = 'Please select your country (so we can make a cool D3 map!).';
+    this.MENTORSHIP_ERROR = 'To complete your mentorship prorgram enrollment, please complete the section above.';
```

Comment:
These are constants, it might be better to extract them out of the component into a different location.